nathan-rob/public/scripts/article.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+var app = app || {};
+
+(function(module) {
```

Comment:
What is module? How would you tell the computer what module is since it's not declared anywhere? Hint: pass in the new object you created above

Reply:
added in argument "app" in IIFE call at bottom of page. app is the argument passed in as "module".